argocd/app-of-apps/values.tmpl.yaml

```diff
@@ -284,6 +290,16 @@ minio:
         hosts:
           - minio.apps.${base_domain}
           - minio.apps.${cluster_name}.${base_domain}
+  buckets:
+    - name: thanos
+      policy: none
+      purge: false
+
+thanos:
+  archives:
+    accessKey: "${minio_access_key}"
```

Comment:
Please don't refer to "minio" in the name of the variables because this code will be also used for s3